src/components/wishlist/EmptyWishlist.jsx

```diff
@@ -0,0 +1,24 @@
+export function EmptyWishlist() {
```

Comment:
No need to create empty Wishlist function. You are Iterating the Wishlist items and when Wishlist is empty then page will be empty. If I misunderstood something then you can correct me.

Reply:
It is a component which is rendered when there isn't any item in the wishlist.